lessons/computation-transformations/langs/en-us/index.adoc

```diff
@@ -0,0 +1,151 @@
+= Computation as a Circle Transformation
+
+@description{Students explore _computation_ as a valid way to transform Circles of Evaluation.}
+
+@lesson-prereqs{translating-words-math}
+
+@keywords{Circle Transformation, computation, equivalence}
+
+[@lesson-intro-table]
+|===
+
+| Lesson Goals
+| Students will be able to...
+
+* Develop equivalent Circle representations of a given expression using Computation as a Circle Transformation.
+* Diagram solving with Circles using Computation.
+
+
+| Student-facing Lesson Goals
+|
+
+* Let's transform Circles using Computation.
+
+
+| Materials
+|[.materials-links]
+@material-links
+
+| Preparation
+|
+* The activity in the first section is most impactful if students can stand up and draw their individual Circles on the whiteboard. Having a handful of markers for student use is encouraged.
+
+| Key Points For The Facilitator
+|
+* "Computation" is the first of several Circle Transformations that students will encounter in this course. It is also a useful solving strategy!
+* This lesson foreshadows and exposes students to additional Circle Transformations (laws of arithmetic) that will be delved later. Keep a running list of the "categories" of Circle Transformations that students spot to refer to down the road.
+|===
+
+== Computation as a Circle Transformation @duration{30 minutes}
+
+=== Overview
+
+Students transform a simple arithmetic expression to create as many equivalent Circles of Evaluation as they can. They then categorize these Circles to consider different possible transformations and to identify _Computation_ as a Circle Transformation.
+
+
+=== Launch
+
+Invite students to take a look at this Circle of Evaluation:
```

Comment:
Curious about this explanation of equivalence and the subsequent request to have students make as many equivalent circles as they can. Not convinced about use of the word equivalence. But, assuming you land on wording that sits well with @schanzer, I would suggest that this activity would make more sense if you started with an expression rather than a circle, then showed students a set of circles that could be used to model the expression and asked them to discuss why each of them worked as a model for the expression, and then gave students an expression which you wanted them to model with as many circles as possible.